lib/irb/debug.rb

```diff
@@ -57,6 +57,19 @@ def DEBUGGER__.capture_frames(*args)
           DEBUGGER__::ThreadClient.prepend(SkipPathHelperForIRB)
         end
 
+        Reline.output_modifier_proc = proc do |output, _|
+          next output if output.strip.empty?
+
+          cmd = output.split(/\s/, 2).first
+
+          if DEBUGGER__.commands.keys.grep(/\A#{Regexp.escape(cmd)}/).any?
+            hint = Color.colorize(" # debug command", [:DIM])
+            output.sub(/\n$/, hint)
+          else
+            output
```

Comment:
Looks like IRB's syntax highlight is disabled here. Can we call IRB's default output_modifier logic here?